Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java

```diff
@@ -95,72 +94,39 @@ public AuditRepositoryOrchImpl() {
-        String eventParticipationIDTypeCode = null;
-        String eventPatientID = null;
-        int eventParticipationTypeCode = 0;
-        int eventParticipationTypeCodeRole = 0;
 
-        List<ActiveParticipant> activeParticipantList = mess.getAuditMessage().getActiveParticipant();
         EventIdentificationType eventIdentification = mess.getAuditMessage().getEventIdentification();
-        List<ParticipantObjectIdentificationType> participantObjectIdentificationList = mess.getAuditMessage()
-            .getParticipantObjectIdentification();
-
-        if (activeParticipantList != null && activeParticipantList.size() > 0) {
-            activeParticipant = (ActiveParticipant) activeParticipantList.get(0);
-            if (activeParticipant != null) {
-                eventUserId = activeParticipant.getUserID();
-                if (eventUserId != null && !eventUserId.equals("")) {
-                    /* this value is temporary set to empty strings and due to length of characters are exceeding
-                     more than 100*/
-                    auditRec.setUserId("");
-                } else {
-                    auditRec.setUserId("");
-                }
-            }
-        }
+
+        auditRec.setUserId("");
 
         eventCommunityId = getCommunityID(mess);
         LOG.info("auditSourceID : " + eventCommunityId);
         if (eventCommunityId != null && !eventCommunityId.equals("")) {
-            auditRec.setCommunityId(eventCommunityId);
+            auditRec.setRemoteHcid(eventCommunityId);
         } else {
-            auditRec.setCommunityId("");
-        }
-
-        if (participantObjectIdentificationList != null && participantObjectIdentificationList.size() > 0) {
-            participantObjectIdentificationType
-                = (ParticipantObjectIdentificationType) participantObjectIdentificationList.get(0);
-            if (participantObjectIdentificationType != null) {
-                eventPatientID = participantObjectIdentificationType.getParticipantObjectID();
-                auditRec.setReceiverPatientId(eventPatientID);
-                eventParticipationTypeCode = participantObjectIdentificationType.getParticipantObjectTypeCode();
-                auditRec.setParticipationTypeCode(eventParticipationTypeCode);
-                eventParticipationTypeCodeRole = participantObjectIdentificationType.getParticipantObjectTypeCodeRole();
-                auditRec.setParticipationTypeCodeRole(eventParticipationTypeCodeRole);
-                eventParticipationIDTypeCode = participantObjectIdentificationType.getParticipantObjectIDTypeCode()
-                    .getCode();
-                auditRec.setParticipationIDTypeCode(eventParticipationIDTypeCode);
-            }
+            auditRec.setRemoteHcid("");
         }
 
-        auditRec.setMessageType(mess.getInterface() + " " + mess.getDirection());
+        auditRec.setDirection(mess.getInterface() + " " + mess.getDirection());
```

Comment:
won't this setDirection be overridden by the one below?  auditRec.setDirection("EventId");